src/util/ped.hpp

```diff
@@ -19,7 +19,19 @@ namespace big::ped
 		PLAYER::SET_PLAYER_MODEL(self::id, hash);
 		script::get_current()->yield();
 		STREAMING::SET_MODEL_AS_NO_LONGER_NEEDED(hash);
-
+		if (ENTITY::GET_ENTITY_MODEL(self::ped) != ENTITY::GET_ENTITY_MODEL(self::ped)) {
+			return false;
+		}
```

Comment:
I might be confused but it looks like this will never be true.

Reply:
Correct since both calls are made on the same tick, it's best to store model from the last tick and compare against that one.